Kernel/UnixTypes.h

```diff
@@ -320,6 +321,12 @@ typedef u32 gid_t;
 typedef u32 clock_t;
 typedef u32 socklen_t;
 typedef int pid_t;
+// Avoid interference with AK/Types.h and LibC/sys/types.h by defining *separate* names:
+TYPEDEF_DISTINCT_NUMERIC_GENERAL(pid_t, false, false, true, false, false, false, ProcessID);
```

Comment:
This macro is very hard to read, can we use enums instead of blind bools?

Reply:
Right, while writing this I thought "we can do (something like this) in the end, when we know what we actually need". And then forgot to do that. Sure, will do.